packages/rs-drive-abci/src/execution/platform_events/core_based_updates/update_quorum_info/v0/mod.rs

```diff
@@ -102,27 +110,39 @@ where
             }
         });
 
-        // Map to quorums
-        let new_quorums = quorum_infos
+        // Map to validator sets
+        let new_validator_sets = quorum_infos
             .into_iter()
-            .map(|(key, info_result)| {
+            .map(|(quorum_hash, info_result)| {
                 let validator_set = ValidatorSet::V0(ValidatorSetV0::try_from_quorum_info_result(
                     info_result,
                     block_platform_state,
                 )?);
-                Ok((key, validator_set))
+
+                tracing::trace!(
+                    ?validator_set,
+                    ?quorum_hash,
+                    quorum_type = ?self.config.quorum_type(),
+                    "add new validator set {} with quorum type {}",
+                    quorum_hash,
+                    self.config.quorum_type()
+                );
+
+                Ok((quorum_hash, validator_set))
             })
             .collect::<Result<Vec<_>, Error>>()?;
+
         // Add new validator_sets entries
         block_platform_state
             .validator_sets_mut()
-            .extend(new_quorums.into_iter());
+            .extend(new_validator_sets);
 
+        // Sort all validator sets into deterministic order by core block height of creation
         block_platform_state
             .validator_sets_mut()
             .sort_by(|_, quorum_a, _, quorum_b| {
                 let primary_comparison = quorum_b.core_height().cmp(&quorum_a.core_height());
-                if primary_comparison == Ordering::Equal {
+                if primary_comparison == std::cmp::Ordering::Equal {
```

Comment:
why std::cmp::?

Reply:
There were usage `std::cmp::Ordering` and just `Ordering` in this module. I made it consistent with the full path: I found it more descriptive.